new_platforms/vsextension/ProjectWizard/WizardImplementation.cs

```diff
@@ -83,6 +83,13 @@ private bool SetOETADevKitPath(Dictionary<string, string> replacementsDictionary
             string unixPath = "/mnt/" + drive + "/" + relativeFolder;
 
             replacementsDictionary.Add("$OETADevKitPath$", unixPath);
+
+            string buildFlavor = board.Split('\\')[0];
+            replacementsDictionary.Add("$OpteeBuildFlavor$", buildFlavor);
```

Comment:
Why is `Split('\\')[0]` necessary?

Also, if users pick their own dev kit, `board` will contain `"Other"` so the replacement of `OpteeBuildFlavor` will yield a path into the NuGet package that won't exist (in `optee\sub.mk`). Come to think of it, if users pick their own dev kit, they should also provide the OE libs, since those are potentially dev kit-specific. This would not have worked before this PR, so in that sense it's not a regression.

Reply:
the split is necessary because the actual strings look like "ls-ls1012grapeboard\\export-ta_arm64" and I only want the first part if there's other path segments.  The strings are in BoardPickerPage.cs starting at line 26.

Reply:
Good point on the "Other", will do that in a future changelist.

Reply:
OK, I'll mark this PR as approved then.